app/migration/migrators/school.rb

```diff
@@ -0,0 +1,79 @@
+module Migrators
+  class School < Migrators::Base
+    FIELDS_MAPPING = {
+      address_line1: :address_line1,
+      address_line2: :address_line2,
+      address_line3: :address_line3,
+      administrative_district_name: :administrative_district_name,
+      funding_eligibility: :funding_eligibility,
+      induction_eligibility: :induction_eligibility,
+      in_england: :in_england?,
+      local_authority_code: :local_authority_code,
+      name: :name,
+      phase_name: :school_phase_name,
+      postcode: :postcode,
+      primary_contact_email: :primary_contact_email,
+      secondary_contact_email: :secondary_contact_email,
+      section_41_approved: :section_41_approved,
+      status: :status,
+      type_name: :school_type_name,
+      ukprn: :ukprn_to_i
+    }.freeze
+
+    MISMATCH_FIELD_MESSAGE = ->(school, field, gias_value, ecf_value) { "School #{school.urn} (#{school.name}) mismatch value on field named '#{field}': #{ecf_value} on ECF whilst #{gias_value} expected on RECT!" }
```

Comment:
Probably a bit easier to read with quotes around the values:

```suggestion
    MISMATCH_FIELD_MESSAGE = ->(school, field, gias_value, ecf_value) { "School #{school.urn} (#{school.name}) mismatch value on field named '#{field}': '#{ecf_value}' on ECF whilst '#{gias_value}' expected on RECT!" }
```